commands/ci_status.go

```diff
@@ -88,15 +88,13 @@ func ciStatus(cmd *Command, args *Args) {
 		response, err := gh.FetchCIStatus(project, sha)
 		utils.Check(err)
 
-		state := response.State
+		state := ""
 		if len(response.Statuses) > 0 {
 			for _, status := range response.Statuses {
 				if checkSeverity(status.State) > checkSeverity(state) {
 					state = status.State
 				}
 			}
-		} else if len(response.Statuses) == 0 {
```

Comment:
Can't we avoid the condition on line 94 since value of `state` is constant before it and convert this block to `else` from `else if`